homeassistant/components/zha/sensor.py

```diff
@@ -313,6 +314,23 @@ async def async_update(self) -> None:
         await super().async_update()
 
 
+@MULTI_MATCH(channel_names=CHANNEL_ELECTRICAL_MEASUREMENT)
+class ElectricalMeasurementApparentPower(
+    ElectricalMeasurement, id_suffix="apparent_power"
+):
+    """Apparent power measurement."""
+
+    SENSOR_ATTR = "apparent_power"
```

Comment:
Do you want to add this attribute for ZigBee attribute reporting in the channel? https://github.com/home-assistant/core/blob/6175f1b6f3f441e3d0708ba3b61dbfdc6ce7e86f/homeassistant/components/zha/core/channels/homeautomation.py#L65

Reply:
Thank you for the pointer, I will add it !